.github/workflows/ci.yaml

```diff
@@ -17,6 +36,13 @@ jobs:
         run: |
           go get ./pinecone
       - name: Run tests
+        continue-on-error: true
```

Comment:
We still want to run the subsequent step that specifically runs local integration tests.